### PR TITLE
[PRM-1676] Update appeals frontend logic to send multiple appeals

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

**Comment by ruthdas03** [comment]
Can you please rebase.

File: app/services/AppealService.scala [hunk 1]
```diff
@@ -108,16 +109,29 @@ class AppealService @Inject()(penaltiesConnector: PenaltiesConnector,
     val appealType = userRequest.answers.getAnswer[PenaltyTypeEnum.Value](SessionKeys.appealType)
     val isLPP = appealType.contains(PenaltyTypeEnum.Late_Payment) || appealType.contains(PenaltyTypeEnum.Additional)
     val agentReferenceNo = userRequest.arn
-    val penaltyNumber = userRequest.answers.getAnswer[String](SessionKeys.penaltyNumber).get
     val correlationId: String = idGenerator.generateUUID
     val userHasUploadedFiles = userRequest.answers.getAnswer[String](SessionKeys.isUploadEvidence).contains("yes")
+    if (userRequest.answers.getAnswer[String](SessionKeys.doYouWantToAppealBothPenalties).contains("yes")) {
+      multipleAppeal(enrolmentKey, appealType, isLPP, correlationId, agentReferenceNo, userHasUploadedFiles, reasonableExcuse)
```
Comment (ruthdas03) [inline app/services/AppealService.scala:115]: 'multipleAppeal' would be called when we have LPP and Additional, I don't think we need this when there's LSP

File: test/services/AppealServiceSpec.scala [hunk 2]
```diff
@@ -50,7 +50,21 @@ class AppealServiceSpec extends SpecBase {
...
+  )))(fakeRequest)
+
+  val fakeRequestForCrimeJourneyMultiple: UserRequest[AnyContent] = UserRequest("123456789", answers = userAnswers(Json.obj(
+    SessionKeys.reasonableExcuse -> "crime",
+    SessionKeys.dateCommunicationSent -> LocalDate.parse("2021-12-01"),
+    SessionKeys.hasCrimeBeenReportedToPolice -> "yes",
+    SessionKeys.hasConfirmedDeclaration -> true,
+    SessionKeys.dateOfCrime -> LocalDate.parse("2022-01-01"),
+    SessionKeys.penaltyNumber -> "123456789",
+    SessionKeys.appealType -> PenaltyTypeEnum.Late_Submission,
```
Comment (ruthdas03) [inline test/services/AppealServiceSpec.scala:64]: Wouldn't this be the case for LPP and Additional

**Review by ruthdas03 — COMMENTED** [review]
Couple of observation on the basis of my understanding that multiple penalties is the case of payment not submission. Please correct me if I'm wrong.

File: app/services/AppealService.scala (hunk 1 above)
Comment (AndrewRButcher) [reply app/services/AppealService.scala:115]: Tightened up logic to discount LSPs

File: test/services/AppealServiceSpec.scala (hunk 2 above)
Comment (AndrewRButcher) [reply test/services/AppealServiceSpec.scala:64]: Updated in accordance with new logic

**Comment by AndrewRButcher** [comment]
test this

**Comment by AndrewRButcher** [comment]
test this please

**Review by ruthdas03 — APPROVED** [review]

**Pull request merged**